tox.ini

```diff
@@ -138,23 +123,13 @@ commands =
     ; Publish coverage reports to coveralls.
     coveralls-push: coveralls
 
-    ; Build in a root folder so that buildbot can pick the result
-    wheel: python -m pip wheel {toxinidir} --no-deps --use-pep517 --wheel-dir {toxinidir}/dist
-
     lint: pre-commit {posargs:run --all-files --show-diff-on-failure}
 
     apidocs: {toxinidir}/bin/admin/build-apidocs {toxinidir}/src/ apidocs
     narrativedocs: sphinx-build -aW -b html -d {toxinidir}/docs/_build {toxinidir}/docs {toxinidir}/docs/_build/
 
     newsfragment: python {toxinidir}/bin/admin/check-newsfragment "{toxinidir}"
 
-    manifest-checker: check-manifest --ignore "docs/_build/**,docs/historic/**,admin/**,bin/admin/**"
-
-    twine: twine check {distdir}/*.*
```

Comment:
This should not have been removed

Reply:
it was not removed, it was moved to `tox release-prepare`